### ISSUE TITLE
Trouble with connecting lxdbr0 to the current bridge.

### ISSUE THREAD

**Issue by technicalflow** [lxc/lxd#10177]
# Required information
lxc info
 * Distribution: OpenSuse Leap
 * Distribution version: 15.3
 * The output of "lxc info" or if that fails:
  driver: lxc | qemu
  driver_version: 4.0.5 | 5.2.0
  firewall: nftables
  kernel: Linux
  kernel_architecture: x86_64
  kernel_features:
    netnsid_getifaddrs: "true"
    seccomp_listener: "true"
    seccomp_listener_continue: "false"
    shiftfs: "false"
    uevent_injection: "true"
    unpriv_fscaps: "true"
  kernel_version: 5.3.18-150300.59.54-default
  lxc_features:
    cgroup2: "true"
    core_scheduling: "false"
    devpts_fd: "true"
    idmapped_mounts_v2: "false"
    mount_injection_file: "true"
    network_gateway_device_route: "true"
    network_ipvlan: "true"
    network_l2proxy: "true"
    network_phys_macvlan_mtu: "true"
    network_veth_router: "true"
    pidfd: "true"
    seccomp_allow_deny_syntax: "true"
    seccomp_notify: "true"
    seccomp_proxy_send_notify_fd: "true"
  os_name: openSUSE Leap
  os_version: "15.3"
  project: default
  server: lxd
  server_clustered: false
  server_name: sv1
  server_pid: 1595
  server_version: "4.21"
  storage: btrfs
  storage_version: 4.19.1
  storage_supported_drivers:
  - name: dir
    version: "1"
    remote: false
  - name: lvm
    version: 2.03.05(2) (2019-06-15) / 1.03.01 (2019-06-15) / 4.42.0
    remote: false
  - name: btrfs
    version: 4.19.1
    remote: false

lxc network show lxdbr0
config:
  ipv4.address: 192.168.25.1/24
  ipv4.nat: "true"
  ipv6.address: fd42:876b:d25e:a88d::1/64
  ipv6.nat: "true"
description: ""
name: lxdbr0
type: bridge
used_by:
  /1.0/profiles/default
managed: true
status: Created
locations:
  none

# Issue description
The only network interface on the server is the eth0 which is connected to bridge0.
After lxd init and creating lxdbr0 bridge, containers are not getting ipv4 address from lxdbr0 and are unable to connect to any network.
Manually creating another lxd network bridge gives the same effect.
I tried adding lxdbr0 to bridge0, but without effect.
Macvlan network with parent set as bridge0 work normally, providing ip from external address pool.

# Steps to reproduce

 1. Step one: create a bridge (nmcli connection add type bridge con-name bridge0 ifname bridge0)
 3. Step two: add eth0 to bridge (nmcli connection modify eth0 master bridge0 && nmcli connection up bridge0)
 4. Step three: install and init lxd

**Comment by stgraber** [comment]
The fact that normal LXD bridges aren't working for you is most likely due to firewalling on your system. This can be caused by ufw or firewalld as well as by having Docker installed on the host system.

Though my first question here would be why aren't you just telling LXD to use your pre-existing bridge rather than using macvlan on top of it.

**Comment by technicalflow** [comment]
Hello, 
Thank you for replying.
My firewall-cmd
```
firewall-cmd --list-all
public (active)
  target: default
  icmp-block-inversion: no
  interfaces: bridge0 eth0
  sources:
  services: dhcpv6-client ssh
  ports:
  protocols:
  forward: no
  masquerade: no
  forward-ports:
  source-ports:
  icmp-blocks:
  rich rules:
```
The bridge0 is configured only to have external IP - it is not able to provide internal IPv4 address to lxc containers. 
When I start a container with bridge0 network I get IP from computer external network (10.10.0.0/24) for container.
Additionally I got virbr1 - which is a bridge created by virtual manager for use with KVM, 
and when I start a container with this network I got the internal (192.168.100.0/24) IP.
